circuit/src/poseidon/params/mod.rs

```diff
@@ -28,20 +33,26 @@ pub trait RoundParams<F: FieldExt, const WIDTH: usize>: Sbox {
 		result
 	}
 
+	/// Returns MDS matrix with a size of WIDTH x WIDTH.
 	fn mds() -> [[F; WIDTH]; WIDTH] {
 		let mds_raw = Self::mds_raw();
 		mds_raw.map(|row| row.map(|item| hex_to_field(item)))
 	}
 
+	/// Returns round constants in its hex string form.
 	fn round_constants_raw() -> Vec<&'static str>;
+	/// Returns MDS martrix in its hex string form.
 	fn mds_raw() -> [[&'static str; WIDTH]; WIDTH];
 }
 
 pub trait Sbox {
+	/// Returns exp^5.
 	fn sbox_expr<F: FieldExt>(exp: Expression<F>) -> Expression<F>;
+	/// Returns f^5.
```

Comment:
These should just say that it applies the Sbox exponentiation. The exponentiation is different for each implementation